website-frontend/eslint.config.js

```diff
@@ -30,6 +30,6 @@ export default tseslint.config(
 		}
 	},
 	{
-		ignores: ['build/', '.svelte-kit/', 'dist/']
+		ignores: ['build/', '.svelte-kit/', 'dist/', 'tests/']
```

Comment:
I think there's no need to ignore the `tests/` folder for linting. For now, we can just ignore the errors themselves as I'll be making dummy tests in a future PR.

Reply:
Or rather, we can leave the unused test files blank so that the linter would not detect any unused imports at all.